.vscode/cspell.json

```diff
@@ -18,8 +22,134 @@
+    "sdk/authorization/arm-authorization/review/*.md",
+    "sdk/compute/arm-compute/review/*.md",
+    "sdk/confidentialledger/confidential-ledger-rest/review/*.md",
+    "sdk/core/core-client-lro-rest/review/*.md",
+    "sdk/core/core-client-paging-rest/review/*.md",
+    "sdk/core/core-client-rest/review/*.md",
+    "sdk/documenttranslator/ai-document-translator-rest/review/*.md",
+    "sdk/eventgrid/arm-eventgrid/review/*.md",
+    "sdk/eventhub/arm-eventhub/review/*.md",
+    "sdk/extendedlocation/arm-extendedlocation/review/*.md",
+    "sdk/features/arm-features/review/*.md",
+    "sdk/keyvault/arm-keyvault/review/*.md",
+    "sdk/links/arm-links/review/*.md",
+    "sdk/locks/arm-locks/review/*.md",
+    "sdk/managedapplications/arm-managedapplications/review/*.md",
+    "sdk/network/arm-network/review/*.md",
+    "sdk/operationalinsights/arm-operationalinsights/review/*.md",
+    "sdk/policy/arm-policy/review/*.md",
+    "sdk/purview/arm-purview/review/*.md",
+    "sdk/purview/purview-account-rest/review/*.md",
+    "sdk/purview/purview-administration-rest/review/*.md",
+    "sdk/purview/purview-catalog-rest/review/*.md",
+    "sdk/purview/purview-scanning-rest/review/*.md",
+    "sdk/quantum/quantum-jobs/review/*.md",
+    "sdk/quota/arm-quota/review/*.md",
+    "sdk/redis/arm-rediscache/review/*.md",
+    "sdk/resources/arm-resources/review/*.md",
+    "sdk/resources-subscriptions/arm-resources-subscriptions/review/*.md",
+    "sdk/security/arm-security/review/*.md",
+    "sdk/servicebus/arm-servicebus/review/*.md",
+    "sdk/sql/arm-sql/review/*.md",
+    "sdk/storage/arm-storage/review/*.md",
+    "sdk/synapse/synapse-access-control/review/*.md",
+    "sdk/synapse/synapse-artifacts/review/*.md",
+    "sdk/synapse/synapse-managed-private-endpoints/review/*.md",
+    "sdk/synapse/synapse-monitoring/review/*.md",
+    "sdk/synapse/synapse-spark/review/*.md",
+    "sdk/templatespecs/arm-templatespecs/review/*.md",
+    "sdk/web-pubsub/arm-webpubsub/review/*.md",
+    "sdk/attestation/attestation/review/**/*.md",
+    "sdk/communication/communication-common/review/**/*.md",
+    "sdk/containerregistry/container-registry/review/**/*.md",
+    "sdk/core/core-amqp/review/**/*.md",
+    "sdk/cosmosdb/cosmos/review/**/*.md",
+    "sdk/digitaltwins/digital-twins-core/review/**/*.md",
+    "sdk/formrecognizer/ai-form-recognizer/review/**/*.md",
+    "sdk/identity/identity/review/**/*.md",
+    "sdk/iot/iot-modelsrepository/review/**/*.md",
+    "sdk/keyvault/keyvault-certificates/review/**/*.md",
+    "sdk/keyvault/keyvault-keys/review/**/*.md",
+    "sdk/monitor/monitor-query/review/**/*.md",
+    "sdk/search/search-documents/review/**/*.md",
+    "sdk/storage/storage-blob/review/**/*.md",
+    "sdk/videoanalyzer/video-analyzer-edge/review/**/*.md"
+  ],
+  "words": [
+    "adfs",
+    "agrifood",
+    "AMQP",
+    "centralus",
+    "CONTOSO",
+    "deps",
+    "deserialization",
+    "devdeps",
+    "eastus",
```

Comment:
Is there no way we can have some sort of base cspell to include all the azure regions?

Reply:
Or maintain our own dictionary in a common place and reference that as a source, instead of putting this all in json?

Reply:
CSpell does support this -- https://cspell.org/docs/dictionaries-custom/  ... We do have some discussions involving shared dictionaries. https://github.com/Azure/azure-sdk-tools/issues/2160